packages/next/client/image.tsx

```diff
@@ -486,40 +486,54 @@ export default function Image({
     imgStyle = undefined
   }
   return (
-    <div style={wrapperStyle}>
-      {shouldPreload
-        ? generatePreload({
-            src,
-            layout,
-            unoptimized,
-            width: widthInt,
-            sizes,
-            quality: qualityInt,
-          })
-        : null}
-      {sizerStyle ? (
-        <div style={sizerStyle}>
-          {sizerSvg ? (
-            <img
-              style={{ maxWidth: '100%', display: 'block' }}
-              alt=""
-              aria-hidden={true}
-              role="presentation"
-              src={`data:image/svg+xml;charset=utf-8,${sizerSvg}`}
-            />
-          ) : null}
-        </div>
-      ) : null}
-      <img
-        {...rest}
-        {...imgAttributes}
-        decoding="async"
-        className={className}
-        sizes={sizes}
-        ref={thisEl}
-        style={imgStyle}
-      />
-    </div>
+    <>
+      <div style={wrapperStyle}>
+        {shouldPreload
+          ? generatePreload({
+              src,
+              layout,
+              unoptimized,
+              width: widthInt,
+              sizes,
+              quality: qualityInt,
+            })
+          : null}
+        {sizerStyle ? (
+          <>
+            <div style={sizerStyle}>
+              {sizerSvg ? (
+                <img
+                  style={{ maxWidth: '100%', display: 'block' }}
+                  alt=""
+                  aria-hidden={true}
+                  role="presentation"
+                  src={`data:image/svg+xml;charset=utf-8,${sizerSvg}`}
+                />
+              ) : null}
+            </div>
+          </>
+        ) : null}
+        <noscript>
+          <img
+            {...rest}
+            src={src}
+            decoding="async"
+            style={{ width: '100%' }}
```

Comment:
```suggestion
            style={imgStyle}
```

The styles should be the same, as the other image. Otherwise `layout` won't be applied properly.